ons_alpha/core/blocks/embeddable.py

```diff
@@ -14,15 +16,47 @@ class Meta:
         template = "templates/components/streamfield/image_block.html"
 
 
+class DocumentBlockStructValue(blocks.StructValue):
+    def as_macro_data(self):
+        return {
+            "thumbnail": True,
+            "url": self["document"].url,
+            "title": self["title"] or self["document"].filename,
```

Comment:
Since document tile is mandatory, fall back to that.
```suggestion
            "title": self["title"] or self["document"].title,
```